packages/ui/src/components/Boilerplate/Boilerplate.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+    ArrowUpRightIcon,
+    DevicePhoneMobileIcon,
+    EnvelopeIcon,
+    GlobeAltIcon,
+    MapPinIcon,
+} from '@heroicons/react/24/outline';
+import type { NewsroomCompanyInformation } from '@prezly/sdk';
+import { hasAnyContactInformation } from '@prezly/theme-kit-core';
+import { twMerge } from 'tailwind-merge';
+
+import { Link } from '../Link';
+
+export interface Props {
+    className?: string;
+    companyInformation: NewsroomCompanyInformation;
+}
+
+export function Boilerplate({ className, companyInformation }: Props) {
+    const { about, phone, email, address, website } = companyInformation;
+    const hasContactInformation = hasAnyContactInformation(companyInformation);
```

Comment:
I think we should not render anything if there is no about information and no contact information.

Reply:
Updated